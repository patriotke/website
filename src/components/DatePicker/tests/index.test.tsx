/* eslint-disable import/no-extraneous-dependencies */
import '@testing-library/jest-dom';
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import { render, screen } from '@testing-library/react';
import DatePicker from '../index';

describe('DatePicker', () => {
  it('renders DatePicker', () => {
    render(<DatePicker />);

    const { container } = render(<DatePicker />);
    expect(container).toMatchSnapshot();
  });
});
