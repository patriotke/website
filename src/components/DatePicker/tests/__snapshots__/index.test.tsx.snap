// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker renders DatePicker 1`] = `
<div>
  <div>
    DatePicker
  </div>
</div>
`;
